cla-backend/cla/models/github_models.py

```
@@ -398,9 +398,9 @@ def oauth2_redirect(self, state, code, request):  # pylint: disable=too-many-arg
             try:
                 token = self._fetch_token(client_id, state, token_url, client_secret, code)
             except Exception as err:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)